service/storage_service.cc

```diff
@@ -2336,6 +2350,7 @@ future<> storage_service::handle_state_normal(inet_address endpoint) {
 
     if (_raft_topology_change_enabled) {
         slogger.debug("ignore handle_state_normal since topology change are using raft");
+        _normal_state_handled_on_boot.insert(endpoint);
```

Comment:
I do not think this is needed any longer.

Reply:
Right